ffi/wgpu.h

```diff
@@ -1,6 +1,6 @@
 #define WGPU_LOCAL
 
-/* Generated with cbindgen:0.9.0 */
+/* Generated with cbindgen:0.8.7 */
```

Comment:
nit: ideally we should try stay on the latest cbindgen version but it's not too important for this change

Reply:
right, I should have seen this